### PR TITLE
Add Bulgarian and pseudo language

### PR DESCRIPTION
**Pull request by camilavcoutinho**
What is the purpose of this pull request?
Add Bulgarian as new language and add pseudo language to implement In-context, a translation tool that gives the translators more context, making it possible for them to translate in a test environment.

### CODE REVIEW

**Review by igorbrasileiro — COMMENTED** [review]
@camilavcoutinho can you share a workspace link? I don't have enough context about it.

**Comment by camilavcoutinho** [comment]
> @camilavcoutinho can you share a workspace link? I don't have enough context about it.

**Comment by camilavcoutinho** [comment]
Hey @igorbrasileiro ! It will only appear when we enable the pseudo language with a script, so don't worry, it will not appear on the UI. The workspace is this one: https://acholi--storecomponents.myvtex.com/admin, but you probably won't have access, because it is linked with Crowdin platform.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- I18n Bg and Ds (pseudo-language to implement in-context tool).
```
Comment (daniserejo) [inline CHANGELOG.md:11]:
```suggestion
- I18n Bg and Bs (pseudo-language to implement in-context tool).
```

**Review by daniserejo — CHANGES_REQUESTED** [review]

**Review by daniserejo — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- I18n Bg and Bs (pseudo-language to implement in-context tool).
+
```
Comment (camilavcoutinho) [inline CHANGELOG.md:12]:
```suggestion

### Fixed
- I18n Es and It.

```

**Review by camilavcoutinho — COMMENTED** [review]
@daniserejo, please remove pt-BR file.

**Review by victorhmp — APPROVED** [review]

**Pull request merged**